File: Crio/crio_programming_interview_problems-master/CountOfSmallerNumbersAfterSelf/CountOfSmallerNumbersAfterSelf.cpp
#include<bits/stdc++.h>
#include "Solution.cpp"
using namespace std;
// CRIO_SOLUTION_START_MODULE_L1_PROBLEMS
// CRIO_SOLUTION_END_MODULE_L1_PROBLEMS
int main() {
    int n;
    cin >> n;
    vector<int>nums(n);
    for(int i = 0 ; i < n ; i++) {
        cin >> nums[i];
    }
    Solution answer;
    vector<int> result = answer.countSmaller(nums);
    for(int i = 0 ; i < result.size() ; i++) {
        cout << result[i] <<" ";
    }
}

File: Crio/crio_programming_interview_problems-master/CountOfSmallerNumbersAfterSelf/CountOfSmallerNumbersAfterSelf.py
from Solution import Solution
import sys
# CRIO_SOLUTION_START_MODULE_L1_PROBLEMS
# CRIO_SOLUTION_END_MODULE_L1_PROBLEMS

n = int(sys.stdin.readline().strip())
nums = list(map(int, sys.stdin.readline().strip().split()))

result = Solution().countSmaller(nums)
sys.stdout.write(' '.join([str(cnt) for cnt in result]) + '\n')

